server/models/handlers/account.js

```diff
@@ -1,22 +1,24 @@
 /* The goal: Take in the user input from the Account Creation form in 
 the front end UI in order to create a user account in the backend*/
 
-async function createUserAccount(firstName, lastName, type, studentID, email, password, phone) {
-    
-    const user = {
-        type: 'student', //note: 'type' is called 'role' in the UI, but refers to either a worker or a student.
-        studentId: '20764242',
-        firstname: 'Victor',
-        lastname: 'Hugo',
-        email: 'victorhugo@gmail.com',
-        phone: '6476442200'
+async function createUserAccount(firstName, lastName, type, studentId, email, password, phone, userType, workerId, specialization, schoolId) {
+    // Searches the student table to see if a student account exists for the given data.
+    var user;
+    const accessToken = new UIDGenerator(UIDGenerator.BASE16, 20);
+    if (userType == userType.student) {
+        //Generates an accessToken of length 20 characters.
```

Comment:
Hi Nancy,
These are super useful comments and I would recommend moving them to be above line 7 as it pertains to that code rather than the below code.

Reply:
Great catch Amy! I've shifted these comments to above the UIDGenerator function!